--- mate-keyring-1.6.1/gcr/mate-gcr.pc.in.orig	2014-01-02 17:31:42.192008002 +0100
+++ mate-keyring-1.6.1/gcr/mate-gcr.pc.in	2014-01-02 17:31:47.345341117 +0100
@@ -11,6 +11,6 @@
 Name: gcr@GCR_VERSION_SUFFIX@
 Description: GObject and GUI library for high level crypto parsing and display
 Version: @VERSION@
-Requires: glib-2.0 gtk+-@GTK_API_VERSION@ libtasn1 mate-gck-0
+Requires: glib-2.0 gtk+-@GTK_API_VERSION@ mate-gck-0
 Libs: -L${libdir} -lmategcr
-Cflags: -I${includedir}/mategcr/gcr
+Cflags: -I${includedir}/mategcr
--- mate-keyring-1.6.0/pkcs11/Makefile.am.orig	2013-03-31 12:28:51.000000000 +0200
+++ mate-keyring-1.6.0/pkcs11/Makefile.am	2013-12-30 10:11:19.893994435 +0100
@@ -1,6 +1,6 @@
 
 # TODO: Are we sure this is the best location for these headers?
-incdir = $(includedir)/gck
+incdir = $(includedir)/mate-gck
 
 inc_HEADERS = \
 	pkcs11.h \
